conf.pri.msvc

```diff
@@ -0,0 +1,40 @@
+# Adapt these paths
+# Point these to the include folders
+INCLUDEPATH += $$quote(C:/qBittorrent/boost_1_51_0)
+INCLUDEPATH += $$quote(C:/qBittorrent/RC_0_16/include)
+INCLUDEPATH += $$quote(C:/qBittorrent/Zlib/include)
+INCLUDEPATH += $$quote(C:/qBittorrent/openssl/include)
+# Point these to the lib folders
+LIBS += $$quote(-LC:/qBittorrent/boost_1_51_0/stage/lib)
+LIBS += $$quote(-LC:/qBittorrent/RC_0_16/bin/path-according-to-the-build-options-chosen)
+LIBS += $$quote(-LC:/qBittorrent/Zlib/lib)
+LIBS += $$quote(-LC:/qBittorrent/openssl/lib)
+
+# Adapt the lib names/versions accordingly
+CONFIG(debug, debug|release) {
+    LIBS += libtorrentd.lib \
+            libboost_system-vc90-mt-sgd-1_51.lib
```

Comment:
Why does this template file list not all required boost librariees?

Reply:
What other libraries are required?
AFAIK, boost.system is only directly required boost library.
In my config I also have the line for boost.datetime (compilation fails without it). Some other libraries (chrono, filesystem, random, thread) are also linked indirectly. I have not delved much, why is it...

Reply:
Guys I have delved into this and see my patch and discussion here: https://github.com/arvidn/libtorrent/pull/2321

Short summary: If we're in c++11 mode only boost.system is needed. However, autotools/bjam doesn't reflect that. But if we ommit them during qbt linking it does work.
PS: I assume libtorrent's cmake is also broken.